src/Theme/Global.elm

```diff
@@ -171,6 +202,15 @@ globalStyles =
                 , width (em 1.0)
                 , marginLeft (em 0.3)
                 ]
+            , withMediaPrint <|
```

Comment:
We've been tending to use `( )` syntax. It's more readable somehow since `|>` and `<|` are so similar, but this is also OK.